2023-06-18 15:41:56,937        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: anonymous
2023-06-18 15:41:58,964        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 15:42:53,370        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 15:42:53,371        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_1
2023-06-18 15:42:53,372        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_2
2023-06-18 15:42:53,374        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 15:42:53,377        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_3
2023-06-18 15:42:53,403        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,405        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687092173.3775995, 'user': {'account_name': 'test_3'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,405        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,406        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 50198), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687092173.3716154, 'user': {'account_name': 'test_1'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,408        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,409        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687092173.3726137, 'user': {'account_name': 'test_2'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,907        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 50198), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,907        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 15:42:53,908        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,909        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 15:42:53,910        DEBUG        client.py                 Запущены процессы
2023-06-18 15:42:53,911        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,911        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 15:42:53,912        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,912        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 15:42:53,913        DEBUG        client.py                 Запущены процессы
2023-06-18 15:42:53,915        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,916        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 15:42:53,917        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,917        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 15:42:53,918        DEBUG        client.py                 Запущены процессы
2023-06-18 15:43:36,538        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}
2023-06-18 15:43:36,544        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 15:43:36,544        INFO         client.py                 Отправлено сообщение для пользователя test_2
2023-06-18 15:43:36,546        DEBUG        client.py                 Была вызвана функция create_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:43:36,703        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции message_from_server
2023-06-18 15:43:36,703        INFO         client.py                 Получено сообщение от пользователя test_3:
Hello
2023-06-18 15:44:16,340        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:16,342        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687092256.3393123, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:16,343        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 15:44:16,783        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 15:44:16,785        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>, 'test_2'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 15:44:16,844        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=416, family=2, type=1, proto=0, laddr=('127.0.0.1', 50199), raddr=('127.0.0.1', 8079)>, 'test_2'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 15:44:23,452        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:23,454        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687092263.4512267, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:23,454        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 15:44:23,921        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 15:44:23,923        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 50200), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 15:44:29,605        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:29,606        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 50198), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687092269.6031377, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 15:44:29,607        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 15:44:30,039        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 15:44:30,041        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 50198), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 15:44:30,109        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 50198), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 16:24:36,896        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 16:24:36,908        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 16:24:36,922        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 16:24:37,407        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_3
2023-06-18 16:24:37,424        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_2
2023-06-18 16:24:37,426        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,428        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687094677.4079187, 'user': {'account_name': 'test_3'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,438        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_1
2023-06-18 16:24:37,441        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,443        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=360, family=2, type=1, proto=0, laddr=('127.0.0.1', 50419), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687094677.4240186, 'user': {'account_name': 'test_2'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,457        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,458        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 16:24:37,458        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,459        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,459        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 16:24:37,460        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687094677.4380956, 'user': {'account_name': 'test_1'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,460        DEBUG        client.py                 Запущены процессы
2023-06-18 16:24:37,968        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=360, family=2, type=1, proto=0, laddr=('127.0.0.1', 50419), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,968        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 16:24:37,969        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,970        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 16:24:37,971        DEBUG        client.py                 Запущены процессы
2023-06-18 16:24:37,971        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,971        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 16:24:37,972        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,973        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 16:24:37,974        DEBUG        client.py                 Запущены процессы
2023-06-18 16:25:22,515        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}
2023-06-18 16:25:22,519        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 16:25:22,519        INFO         client.py                 Отправлено сообщение для пользователя test_3
2023-06-18 16:25:22,521        DEBUG        client.py                 Была вызвана функция create_message c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:22,703        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции message_from_server
2023-06-18 16:25:22,703        INFO         client.py                 Получено сообщение от пользователя test_1:
Hello World
2023-06-18 16:25:33,637        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:33,639        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687094733.636207, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:33,639        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 16:25:33,932        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 16:25:33,934        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 16:25:34,141        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=388, family=2, type=1, proto=0, laddr=('127.0.0.1', 50418), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 16:25:41,885        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:41,886        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=360, family=2, type=1, proto=0, laddr=('127.0.0.1', 50419), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687094741.883918, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:41,887        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 16:25:42,078        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 16:25:42,080        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=360, family=2, type=1, proto=0, laddr=('127.0.0.1', 50419), raddr=('127.0.0.1', 8079)>, 'test_2'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 16:25:42,389        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=360, family=2, type=1, proto=0, laddr=('127.0.0.1', 50419), raddr=('127.0.0.1', 8079)>, 'test_2'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 16:25:48,509        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:48,511        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687094748.508022, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 16:25:48,511        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 16:25:48,735        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 16:25:48,737        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=312, family=2, type=1, proto=0, laddr=('127.0.0.1', 50420), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 19:09:26,471        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 19:09:26,480        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 19:09:26,499        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 19:09:28,495        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 19:09:28,510        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 19:09:28,526        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 19:38:48,066        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 19:38:48,129        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 19:38:50,097        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 19:38:50,145        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 19:45:26,887        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 19:45:26,889        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_1
2023-06-18 19:45:26,896        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 19:45:26,897        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_2
2023-06-18 19:45:26,910        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,911        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687106726.8898008, 'user': {'account_name': 'test_1'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,919        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,920        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 51128), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687106726.897781, 'user': {'account_name': 'test_2'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,922        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 19:45:26,923        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_3
2023-06-18 19:45:26,938        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,940        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687106726.9237528, 'user': {'account_name': 'test_3'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,942        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,943        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 19:45:26,947        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,952        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 51128), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,952        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 19:45:26,954        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 19:45:26,954        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:26,956        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 19:45:26,957        DEBUG        client.py                 Запущены процессы
2023-06-18 19:45:26,957        DEBUG        client.py                 Запущены процессы
2023-06-18 19:45:27,472        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:27,472        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 19:45:27,473        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:45:27,487        INFO         client.py                 Принят ответ от сервера 200 : OK
2023-06-18 19:45:27,488        DEBUG        client.py                 Запущены процессы
2023-06-18 19:46:07,830        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687106767.830598, 'mess_text': 'My friends'}
2023-06-18 19:46:07,833        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687106767.830598, 'mess_text': 'My friends'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 19:46:07,834        INFO         client.py                 Отправлено сообщение для пользователя test_3
2023-06-18 19:46:07,835        DEBUG        client.py                 Была вызвана функция create_message c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:08,248        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции message_from_server
2023-06-18 19:46:08,250        INFO         client.py                 Получено сообщение от пользователя test_1:
My friends
2023-06-18 19:46:36,168        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:36,170        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687106796.1666088, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:36,171        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 19:46:36,215        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 19:46:36,216        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 19:46:36,672        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=384, family=2, type=1, proto=0, laddr=('127.0.0.1', 51127), raddr=('127.0.0.1', 8079)>, 'test_1'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 19:46:42,880        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:42,882        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687106802.8793113, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:42,883        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 19:46:43,322        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 19:46:43,323        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 19:46:43,384        DEBUG        threading.py              Была вызвана функция user_interactive c параметрами (<socket.socket fd=296, family=2, type=1, proto=0, laddr=('127.0.0.1', 51129), raddr=('127.0.0.1', 8079)>, 'test_3'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 19:46:49,945        DEBUG        client.py                 Была вызвана функция create_exit_message c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:49,946        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 51128), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687106809.9426124, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции user_interactive
2023-06-18 19:46:49,947        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 19:46:49,954        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 19:46:49,955        DEBUG        threading.py              Была вызвана функция message_from_server c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 51128), raddr=('127.0.0.1', 8079)>, 'test_2'), {}. Вызов из модуля __main__. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:06:30,143        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 21:06:30,144        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_1
2023-06-18 21:06:30,144        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 21:06:30,144        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 21:06:30,145        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_3
2023-06-18 21:06:30,166        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,167        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=320, family=2, type=1, proto=0, laddr=('127.0.0.1', 51306), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687111590.145506, 'user': {'account_name': 'test_2'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,167        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,168        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=332, family=2, type=1, proto=0, laddr=('127.0.0.1', 51305), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687111590.1445096, 'user': {'account_name': 'test_1'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,168        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=396, family=2, type=1, proto=0, laddr=('127.0.0.1', 51307), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687111590.145506, 'user': {'account_name': 'test_3'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,673        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=332, family=2, type=1, proto=0, laddr=('127.0.0.1', 51305), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,673        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:06:30,674        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,674        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:06:30,677        DEBUG        client.py                 Запущены процессы
2023-06-18 21:06:30,677        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=320, family=2, type=1, proto=0, laddr=('127.0.0.1', 51306), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,677        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:06:30,678        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,679        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:06:30,680        DEBUG        client.py                 Запущены процессы
2023-06-18 21:06:30,681        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=396, family=2, type=1, proto=0, laddr=('127.0.0.1', 51307), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,681        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:06:30,682        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:06:30,682        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:06:30,683        DEBUG        client.py                 Запущены процессы
2023-06-18 21:07:16,047        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_1', 'to': 'test_2', 'time': 1687111636.0479417, 'mess_text': 'Hi friends'}
2023-06-18 21:07:16,051        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=332, family=2, type=1, proto=0, laddr=('127.0.0.1', 51305), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_1', 'to': 'test_2', 'time': 1687111636.0479417, 'mess_text': 'Hi friends'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 21:07:16,051        INFO         client.py                 Отправлено сообщение для пользователя test_2
2023-06-18 21:07:16,448        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=320, family=2, type=1, proto=0, laddr=('127.0.0.1', 51306), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:07:16,449        INFO         client.py                 Получено сообщение от пользователя test_1:
Hi friends
2023-06-18 21:08:01,384        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_2', 'to': 'test_3', 'time': 1687111681.3847148, 'mess_text': 'Hello'}
2023-06-18 21:08:01,385        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=320, family=2, type=1, proto=0, laddr=('127.0.0.1', 51306), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_2', 'to': 'test_3', 'time': 1687111681.3847148, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 21:08:01,385        INFO         client.py                 Отправлено сообщение для пользователя test_3
2023-06-18 21:08:01,712        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=396, family=2, type=1, proto=0, laddr=('127.0.0.1', 51307), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:08:01,713        INFO         client.py                 Получено сообщение от пользователя test_2:
Hello
2023-06-18 21:08:51,400        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_3', 'to': 'test_1', 'time': 1687111731.4005957, 'mess_text': 'goodbye'}
2023-06-18 21:08:51,401        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=396, family=2, type=1, proto=0, laddr=('127.0.0.1', 51307), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_3', 'to': 'test_1', 'time': 1687111731.4005957, 'mess_text': 'goodbye'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 21:08:51,402        INFO         client.py                 Отправлено сообщение для пользователя test_1
2023-06-18 21:08:51,565        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=332, family=2, type=1, proto=0, laddr=('127.0.0.1', 51305), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:08:51,565        INFO         client.py                 Получено сообщение от пользователя test_3:
goodbye
2023-06-18 21:08:59,681        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=396, family=2, type=1, proto=0, laddr=('127.0.0.1', 51307), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687111739.6808212, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:08:59,682        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:09:00,174        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 21:09:06,705        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=320, family=2, type=1, proto=0, laddr=('127.0.0.1', 51306), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687111746.703971, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:09:06,706        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:09:06,792        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 21:09:13,457        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=332, family=2, type=1, proto=0, laddr=('127.0.0.1', 51305), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687111753.456341, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:09:13,458        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:09:13,916        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 21:51:26,500        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 21:51:26,511        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 21:51:26,548        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 21:51:28,552        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 21:51:28,599        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-18 21:56:21,855        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-18 21:56:21,856        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_1
2023-06-18 21:56:21,861        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-18 21:56:21,862        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_2
2023-06-18 21:56:21,880        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_1',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,881        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 52604), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687114581.8568363, 'user': {'account_name': 'test_1'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,885        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_2',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,886        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-18 21:56:21,886        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=308, family=2, type=1, proto=0, laddr=('127.0.0.1', 52605), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687114581.8628204, 'user': {'account_name': 'test_2'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,887        DEBUG        client.py                 Сформировано presence сообщение для пользователя test_3
2023-06-18 21:56:21,903        DEBUG        client.py                 Была вызвана функция create_presence c параметрами ('test_3',), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,904        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 52606), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687114581.8868074, 'user': {'account_name': 'test_3'}}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,906        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 52604), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,906        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:56:21,907        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,907        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:56:21,909        DEBUG        client.py                 Запущены процессы
2023-06-18 21:56:21,909        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=308, family=2, type=1, proto=0, laddr=('127.0.0.1', 52605), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,910        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:56:21,910        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:21,911        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:56:21,912        DEBUG        client.py                 Запущены процессы
2023-06-18 21:56:22,421        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 52606), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:22,421        DEBUG        client.py                 Получено сообщение от сервера: {'response': 200}
2023-06-18 21:56:22,422        DEBUG        client.py                 Была вызвана функция process_ans c параметрами ({'response': 200},), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 21:56:22,422        INFO         client.py                 Установлено соединение с сервером.Принят ответ от сервера 200 : OK
2023-06-18 21:56:22,424        DEBUG        client.py                 Запущены процессы
2023-06-18 21:57:18,409        DEBUG        client.py                 Сформирован словарь сообщения: {'action': 'message', 'from': 'test_2', 'to': 'test_1', 'time': 1687114638.4094841, 'mess_text': 'goodmorning'}
2023-06-18 21:57:18,411        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=308, family=2, type=1, proto=0, laddr=('127.0.0.1', 52605), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test_2', 'to': 'test_1', 'time': 1687114638.4094841, 'mess_text': 'goodmorning'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции create_message
2023-06-18 21:57:18,411        INFO         client.py                 Отправлено сообщение для пользователя test_1
2023-06-18 21:57:18,458        DEBUG        client.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 52604), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:57:18,458        INFO         client.py                 Получено сообщение от пользователя test_2:
goodmorning
2023-06-18 21:57:24,002        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=308, family=2, type=1, proto=0, laddr=('127.0.0.1', 52605), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687114644.0007727, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:57:24,002        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:57:24,054        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 21:57:29,770        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=408, family=2, type=1, proto=0, laddr=('127.0.0.1', 52604), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687114649.7693386, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:57:29,771        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:57:30,174        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-18 21:57:34,675        DEBUG        client.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 52606), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687114654.6736925, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-18 21:57:34,675        INFO         client.py                 Завершение работы по команде пользователя.
2023-06-18 21:57:34,763        CRITICAL     client.py                 Потеряно соединение с сервером.
2023-06-24 22:23:20,451        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_2
2023-06-24 22:23:20,454        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_1
2023-06-24 22:23:20,455        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1, порт: 8079, имя пользователя: test_3
2023-06-24 22:23:22,494        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-24 22:23:22,510        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-24 22:23:22,510        CRITICAL     client.py                 Не удалось подключиться к серверу 127.0.0.1:8079, конечный компьютер отверг запрос на подключение.
2023-06-24 23:01:49,266        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-24 23:02:03,003        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: test_12
2023-06-25 11:35:22,656        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-25 11:35:40,490        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: RedMaster
2023-06-28 21:06:57,583        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 21:07:14,897        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: Test_12
2023-06-28 21:07:14,972        INFO         transport.py              Попытка подключения №1
2023-06-28 21:07:17,999        INFO         transport.py              Попытка подключения №2
2023-06-28 21:07:21,054        INFO         transport.py              Попытка подключения №3
2023-06-28 21:07:24,103        INFO         transport.py              Попытка подключения №4
2023-06-28 21:07:27,146        INFO         transport.py              Попытка подключения №5
2023-06-28 21:07:30,183        CRITICAL     transport.py              Не удалось установить соединение с сервером
2023-06-28 21:08:08,113        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 21:08:22,668        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: test_21
2023-06-28 21:08:22,715        INFO         transport.py              Попытка подключения №1
2023-06-28 21:08:22,716        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 21:08:22,716        DEBUG        transport.py              Сформировано presence сообщение для пользователя test_21
2023-06-28 21:08:22,718        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687975702.7162752, 'user': {'account_name': 'test_21'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:08:22,744        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:08:22,745        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 21:08:22,745        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 21:08:22,745        DEBUG        transport.py              Запрос списка известных пользователей test_21
2023-06-28 21:08:22,746        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687975702.7455401, 'account_name': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:08:23,260        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:08:23,273        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 21:08:23,273        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687975703.2735238, 'user': 'test_21'}
2023-06-28 21:08:23,274        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687975703.2735238, 'user': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:08:23,770        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:08:23,770        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 21:08:23,771        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 21:08:42,230        DEBUG        transport.py              Запрос списка известных пользователей test_21
2023-06-28 21:08:42,231        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687975722.2303445, 'account_name': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 21:08:42,568        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 21:08:42,583        DEBUG        add_contact.py            Обновление списка пользователей с сервера выполнено
2023-06-28 21:08:44,160        DEBUG        transport.py              Создание контакта client_2
2023-06-28 21:08:44,162        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687975724.1607878, 'user': 'test_21', 'account_name': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 21:09:00,870        DEBUG        transport.py              Запрос списка известных пользователей test_21
2023-06-28 21:09:00,871        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687975740.8708577, 'account_name': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 21:09:05,882        DEBUG        transport.py              Создание контакта client_4
2023-06-28 21:09:06,396        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687975745.8826785, 'user': 'test_21', 'account_name': 'client_4'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 21:09:33,220        DEBUG        transport.py              Создание контакта client_1
2023-06-28 21:09:33,222        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1260, family=2, type=1, proto=0, laddr=('127.0.0.1', 49866), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687975773.2203228, 'user': 'test_21', 'account_name': 'client_1'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 21:11:02,724        CRITICAL     transport.py              Потеряно соединение с сервером.
2023-06-28 21:32:46,631        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 21:32:57,248        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: client_2
2023-06-28 21:32:57,297        INFO         transport.py              Попытка подключения №1
2023-06-28 21:32:57,298        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 21:32:57,298        DEBUG        transport.py              Сформировано presence сообщение для пользователя client_2
2023-06-28 21:32:57,300        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687977177.2984731, 'user': {'account_name': 'client_2'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:32:57,320        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:32:57,320        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 21:32:57,320        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 21:32:57,320        DEBUG        transport.py              Запрос списка известных пользователей client_2
2023-06-28 21:32:57,321        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687977177.3203857, 'account_name': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:32:57,827        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:32:57,837        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 21:32:57,837        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687977177.8372886, 'user': 'client_2'}
2023-06-28 21:32:57,839        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687977177.8372886, 'user': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:32:58,337        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:32:58,337        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 21:32:58,338        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 21:33:47,274        DEBUG        transport.py              Запрос списка известных пользователей client_2
2023-06-28 21:33:47,276        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687977227.2741125, 'account_name': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 21:33:47,689        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 21:33:47,755        DEBUG        add_contact.py            Обновление списка пользователей с сервера выполнено
2023-06-28 21:33:48,679        DEBUG        transport.py              Создание контакта test_21
2023-06-28 21:33:48,681        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687977228.680785, 'user': 'client_2', 'account_name': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 21:34:16,838        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1228, family=2, type=1, proto=0, laddr=('127.0.0.1', 50034), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687977256.7708092, 'account_name': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 21:34:16,838        DEBUG        transport.py              Транспорт завершает работу.
2023-06-28 21:58:33,359        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 21:58:46,920        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: client_12
2023-06-28 21:58:46,974        INFO         transport.py              Попытка подключения №1
2023-06-28 21:58:50,014        INFO         transport.py              Попытка подключения №2
2023-06-28 21:58:53,045        INFO         transport.py              Попытка подключения №3
2023-06-28 21:58:56,060        INFO         transport.py              Попытка подключения №4
2023-06-28 21:58:59,112        INFO         transport.py              Попытка подключения №5
2023-06-28 21:59:02,150        CRITICAL     transport.py              Не удалось установить соединение с сервером
2023-06-28 21:59:18,532        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 21:59:27,640        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: client_12
2023-06-28 21:59:27,672        INFO         transport.py              Попытка подключения №1
2023-06-28 21:59:27,672        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 21:59:27,673        DEBUG        transport.py              Сформировано presence сообщение для пользователя client_12
2023-06-28 21:59:27,674        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687978767.6737812, 'user': {'account_name': 'client_12'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:59:27,698        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 21:59:27,699        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 21:59:27,699        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 21:59:27,699        DEBUG        transport.py              Запрос списка известных пользователей client_12
2023-06-28 21:59:27,701        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687978767.6991098, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:59:28,214        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 21:59:28,224        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 21:59:28,224        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687978768.2245595, 'user': 'client_12'}
2023-06-28 21:59:28,226        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687978768.2245595, 'user': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:59:28,728        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 21:59:28,728        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 21:59:28,728        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 22:00:16,254        DEBUG        transport.py              Создание контакта client_4
2023-06-28 22:00:16,256        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687978816.2549412, 'user': 'client_12', 'account_name': 'client_4'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:01:15,148        DEBUG        transport.py              Создание контакта test_21
2023-06-28 22:01:15,150        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687978875.1483486, 'user': 'client_12', 'account_name': 'test_21'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:02:00,499        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1204, family=2, type=1, proto=0, laddr=('127.0.0.1', 50451), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687978920.4968665, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 22:02:00,499        DEBUG        transport.py              Транспорт завершает работу.
2023-06-28 22:05:06,260        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 22:05:16,301        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: client_12
2023-06-28 22:05:16,333        INFO         transport.py              Попытка подключения №1
2023-06-28 22:05:16,358        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 22:05:16,358        DEBUG        transport.py              Сформировано presence сообщение для пользователя client_12
2023-06-28 22:05:16,360        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687979116.3583543, 'user': {'account_name': 'client_12'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:05:16,863        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:05:16,863        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:05:16,863        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 22:05:16,863        DEBUG        transport.py              Запрос списка известных пользователей client_12
2023-06-28 22:05:16,864        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687979116.8636813, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:05:17,373        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:05:17,382        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 22:05:17,382        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687979117.3826504, 'user': 'client_12'}
2023-06-28 22:05:17,384        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687979117.3826504, 'user': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:05:17,889        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:05:17,889        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 22:05:17,889        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 22:05:27,559        DEBUG        transport.py              Создание контакта client_4
2023-06-28 22:05:27,562        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687979127.5609603, 'user': 'client_12', 'account_name': 'client_4'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:05:45,147        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=840, family=2, type=1, proto=0, laddr=('127.0.0.1', 50590), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687979144.9609156, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 22:05:45,147        DEBUG        transport.py              Транспорт завершает работу.
2023-06-28 22:07:59,615        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 22:08:06,772        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: client_12
2023-06-28 22:08:06,804        INFO         transport.py              Попытка подключения №1
2023-06-28 22:08:06,805        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 22:08:06,805        DEBUG        transport.py              Сформировано presence сообщение для пользователя client_12
2023-06-28 22:08:06,807        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687979286.805017, 'user': {'account_name': 'client_12'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:08:06,822        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:08:06,823        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:08:06,823        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 22:08:06,823        DEBUG        transport.py              Запрос списка известных пользователей client_12
2023-06-28 22:08:06,825        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687979286.8239632, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:08:07,337        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:08:07,347        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 22:08:07,347        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687979287.347157, 'user': 'client_12'}
2023-06-28 22:08:07,349        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687979287.347157, 'user': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:08:07,856        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:08:07,856        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 22:08:07,856        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 22:08:14,781        DEBUG        transport.py              Создание контакта client_2
2023-06-28 22:08:14,784        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687979294.7829235, 'user': 'client_12', 'account_name': 'client_2'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:08:15,019        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:08:15,019        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:08:15,029        INFO         main_window.py            Успешно добавлен контакт <PyQt5.QtGui.QStandardItem object at 0x000002936397C1F0>
2023-06-28 22:08:49,536        DEBUG        transport.py              Сформирован словарь сообщения: {'action': 'message', 'from': 'client_12', 'to': 'client_2', 'time': 1687979329.5362735, 'mess_text': 'Hi, friends!'}
2023-06-28 22:08:49,538        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'client_12', 'to': 'client_2', 'time': 1687979329.5362735, 'mess_text': 'Hi, friends!'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции send_message
2023-06-28 22:09:43,969        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=1268, family=2, type=1, proto=0, laddr=('127.0.0.1', 50658), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687979383.9686704, 'account_name': 'client_12'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 22:09:43,969        DEBUG        transport.py              Транспорт завершает работу.
2023-06-28 22:17:15,082        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 22:17:15,091        DEBUG        client.py                 Была вызвана функция arg_parser c параметрами (), {}. Вызов из модуля __main__. Вызов из функции arg_parser().Вызов из функции <module>
2023-06-28 22:17:15,096        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: test1
2023-06-28 22:17:15,106        INFO         client.py                 Запущен клиент с параметрами: адрес сервера: 127.0.0.1 , порт: 8079, имя пользователя: test2
2023-06-28 22:17:15,140        INFO         transport.py              Попытка подключения №1
2023-06-28 22:17:15,141        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 22:17:15,141        DEBUG        transport.py              Сформировано presence сообщение для пользователя test1
2023-06-28 22:17:15,143        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687979835.1414125, 'user': {'account_name': 'test1'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:17:15,148        INFO         transport.py              Попытка подключения №1
2023-06-28 22:17:15,149        DEBUG        transport.py              Установлено соединение с сервером
2023-06-28 22:17:15,149        DEBUG        transport.py              Сформировано presence сообщение для пользователя test2
2023-06-28 22:17:15,151        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'presence', 'time': 1687979835.1494694, 'user': {'account_name': 'test2'}}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:17:15,171        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:17:15,171        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:17:15,171        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 22:17:15,171        DEBUG        transport.py              Запрос списка известных пользователей test1
2023-06-28 22:17:15,172        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687979835.1716423, 'account_name': 'test1'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:17:15,700        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:17:15,711        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 22:17:15,711        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687979835.7113295, 'user': 'test1'}
2023-06-28 22:17:15,713        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687979835.7113295, 'user': 'test1'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:17:15,718        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции connection_init
2023-06-28 22:17:15,718        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:17:15,718        INFO         transport.py              Соединение с сервером успешно установлено.
2023-06-28 22:17:15,718        DEBUG        transport.py              Запрос списка известных пользователей test2
2023-06-28 22:17:15,720        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687979835.7184284, 'account_name': 'test2'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:17:16,241        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:17:16,241        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 22:17:16,242        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 22:17:16,246        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции user_list_update
2023-06-28 22:17:16,253        DEBUG        transport.py              Запрос контакт листа для пользователя Thread-1
2023-06-28 22:17:16,253        DEBUG        transport.py              Сформирован запрос {'action': 'get_contacts', 'time': 1687979836.2537394, 'user': 'test2'}
2023-06-28 22:17:16,255        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'get_contacts', 'time': 1687979836.2537394, 'user': 'test2'}), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:17:16,758        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_name).Вызов из функции contacts_list_update
2023-06-28 22:17:16,758        DEBUG        transport.py              Получен ответ {'response': 202, 'data_list': []}
2023-06-28 22:17:16,758        DEBUG        transport.py              Запущен процесс - приёмник собщений с сервера.
2023-06-28 22:17:40,366        DEBUG        transport.py              Запрос списка известных пользователей test1
2023-06-28 22:17:40,367        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687979860.3668902, 'account_name': 'test1'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 22:17:40,771        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 22:17:40,782        DEBUG        add_contact.py            Обновление списка пользователей с сервера выполнено
2023-06-28 22:17:43,540        DEBUG        transport.py              Создание контакта test2
2023-06-28 22:17:43,542        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687979863.5400882, 'user': 'test1', 'account_name': 'test2'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:17:43,868        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:17:43,869        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:17:43,879        INFO         main_window.py            Успешно добавлен контакт <PyQt5.QtGui.QStandardItem object at 0x000001DFADDC4280>
2023-06-28 22:17:52,848        DEBUG        transport.py              Создание контакта test1
2023-06-28 22:17:52,953        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'add', 'time': 1687979872.848317, 'user': 'test2', 'account_name': 'test1'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:17:53,071        DEBUG        transport.py              Была вызвана функция get_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>,), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции add_contact
2023-06-28 22:17:53,071        DEBUG        transport.py              Разбор сообщения от сервера: {'response': 200}
2023-06-28 22:17:53,081        INFO         main_window.py            Успешно добавлен контакт <PyQt5.QtGui.QStandardItem object at 0x000001A06F2CC280>
2023-06-28 22:18:13,254        DEBUG        transport.py              Сформирован словарь сообщения: {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687979893.254608, 'mess_text': 'Hi, my friends!'}
2023-06-28 22:18:13,598        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687979893.254608, 'mess_text': 'Hi, my friends!'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции send_message
2023-06-28 22:18:36,392        DEBUG        transport.py              Сформирован словарь сообщения: {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687979916.392835, 'mess_text': 'Hi'}
2023-06-28 22:18:36,393        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687979916.392835, 'mess_text': 'Hi'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции send_message
2023-06-28 22:21:01,196        DEBUG        transport.py              Сформирован словарь сообщения: {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1687980061.1966188, 'mess_text': 'Hello'}
2023-06-28 22:21:01,582        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test2', 'to': 'test1', 'time': 1687980061.1966188, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции send_message
2023-06-28 22:21:13,970        DEBUG        transport.py              Запрос списка известных пользователей test2
2023-06-28 22:21:13,971        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'get_users', 'time': 1687980073.9709392, 'account_name': 'test2'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции user_list_update
2023-06-28 22:21:40,256        DEBUG        transport.py              Сформирован словарь сообщения: {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687980100.256434, 'mess_text': 'Hello'}
2023-06-28 22:21:40,258        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687980100.256434, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции client_app.exec_().Вызов из функции send_message
2023-06-28 22:22:30,460        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=972, family=2, type=1, proto=0, laddr=('127.0.0.1', 50864), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687980150.458233, 'account_name': 'test2'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 22:22:30,460        DEBUG        transport.py              Транспорт завершает работу.
2023-06-28 22:22:34,077        DEBUG        transport.py              Была вызвана функция send_message c параметрами (<socket.socket fd=324, family=2, type=1, proto=0, laddr=('127.0.0.1', 50863), raddr=('127.0.0.1', 8079)>, {'action': 'exit', 'time': 1687980153.6866465, 'account_name': 'test1'}), {}. Вызов из модуля utils. Вызов из функции transport.transport_shutdown().Вызов из функции transport_shutdown
2023-06-28 22:22:34,077        DEBUG        transport.py              Транспорт завершает работу.
